app/assets/javascripts/discourse/app/components/horizontal-overflow-nav.js

```diff
@@ -107,11 +109,10 @@ export default class HorizontalOverflowNav extends Component {
       siblingTarget = event.target.nextElementSibling;
     }
 
+    siblingTarget.scrollLeft += scrollSpeed;
```

Comment:
On click we want to instantly scroll and not have to depend on the interval below.